lib/runBlock.js

```diff
@@ -84,7 +85,7 @@ module.exports = function (opts, cb) {
     function processTx (tx, cb) {
       var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).cmp(new BN(tx.gasLimit).add(gasUsed)) === -1
       if (gasLimitIsHigherThanBlock) {
-        cb('tx has a higher gas limit than the block')
+        cb(new Error('tx has a higher gas limit than the block'))
```

Comment:
oh, right, just a best practice, let me know if you want me to remove it from this PR.